tests/test_router_bgp.rb

```diff
@@ -43,12 +52,14 @@ def create_bgp_vrf(asnum, vrf)
 end
 
 def setup_default
+  @device.cmd('no router bgp')
```

Comment:
Yes it works.

RP/0/RP0/CPU0:ios#sh run router bgp
Mon Nov 23 21:46:06.098 UTC
router bgp 55
 bgp confederation peers
  11
  12
  13
 !
 bgp router-id 1.1.1.1
 bgp log neighbor changes disable
 vrf blue
  rd auto
  bgp router-id 192.168.0.66
 !
!

RP/0/RP0/CPU0:ios#conf
Mon Nov 23 21:46:09.679 UTC

RP/0/RP0/CPU0:ios(config)#no router bgp
RP/0/RP0/CPU0:ios(config)#commit
Mon Nov 23 21:46:17.583 UTC
RP/0/RP0/CPU0:ios(config)#sh run router bgp
Mon Nov 23 21:46:33.733 UTC
% No such configuration item(s)